### DISCUSSION TITLE
enable/disable one of the default actions in Table

### DISCUSSION THREAD

**Discussion by mshanak** [filamentphp/filament#1154] [General]
hello,

Is there a way to disable one of the default actions that come with the table by default?
for example, I need to disable or hide the `delete` action in the many-to-many table.

![Screen Shot 2022-01-06 at 5 30 35 PM](https://user-images.githubusercontent.com/964266/148407485-c63e5204-27e2-4c2e-a879-0afca6d897c5.png)

I hope we will have an option to disable/enable these buttons.

Regards

**Comment by danharrin** [comment]
Try this:

```php
protected function canDelete(Model $record): bool
{
    return false;
}
```
